example/src/example.c

```diff
@@ -1,13 +1,19 @@
 /* example program showing how to use 3dxdisp */
+#include <stdio.h>
 #include "3dxdisp.h"
 #include "logo.h"
 
 
 int main(void)
 {
+	/*
 	if(tdx_open() == -1) {
 		return 1;
 	}
+	*/
+	if (tdx_open_file(NULL) == -1) {
+		return 1;
+	}
```

Comment:
I'd leave the original `tdx_open` call in the example for simplicity, see next comment.